php/php.latte/test/unit/src/org/netbeans/modules/php/latte/indent/LatteIndenterTest.java

```diff
@@ -40,22 +40,6 @@ public void testIfElseBlock() throws Exception {
         indent("testIfElseBlock");
     }
 
-    public void testRealFile_01() throws Exception {
```

Comment:
Well, should uncomment them if test data doesn't exist with the reason.

Reply:
Test files were not part of donation.
Similar problem is in Symfony and Symfony 2 modules.
I sent question to dev@ about it.

Reply:
> Test files were not part of donation.

I know it. Some files were removed on the donation review of Oracle. Because of something problem.

> I sent question to dev@ about it.

Please add it to the description (some people might not check all emails).
It would be nice if you can add the link for the thread.

I've replied.

Reply:
It is definitely pity that these files are not donated. They were tested from some reason.

Reply:
Files in Oracle repository are here: https://hg.netbeans.org/releases/file/5fd841261bf9/php.latte/test/unit/data/testfiles/indent
They were not donated probably because they weren't from Oracle - copyright says author is Ondřej Brejla.

I'm not sure what exactly they were testing. Maybe replacing them with newly created sample files can be enough.

Reply:
[Ondra](https://github.com/obrejla) was my colleague working on the PHP support in NetBeans so - it is weird :)

@obrejla Ondro, could we simply use those files and add them in this review? Don't you remember/know the origin of the files? :) Thanks!

Reply:
Hehe, it's a real file from my Eciovni package https://github.com/obrejla/Eciovni ...You can definitely use it for testing :)

Reply:
And imho it was testing that the parser will not break the parsing loop on a real-world valid file. But you know...it's 10 years already :man_shrugging:

Reply:
@KacerCZ

So, please, could you add those files back there? We have Ondra's agreement here, I hope this could be OK. Thanks!

CC @obrejla @junichi11

Reply:
I tried it yesterday, but failed with file encoding problems.
See https://lists.apache.org/thread.html/ra565c8b965dada529eaed747a2e0ef1aa3aedae8fdc02b570550cdfc%40%3Cdev.netbeans.apache.org%3E

Reply:
This advice does not work?

> For testing purposes, I'd suggest to place a FileEncodingQueryImpl into 
MockLookup in the tests, as here:
org.openide.text.DataEditorSupportEncodingTest

Reply:
It worked, so I updated PR and added two real lile tests.